DOCS-6594 WebSockets Connector (#464)
mulesoft/docs-connectors: modules/ROOT/pages/websocket/websocket-connector-reconnection.adoc

= WebSocket Reconnection
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Web sockets are based on persistent and stateful connections. A connection can abnormally close in the lapse between when it was established and its use in an operation due to factors like idle timeouts, network conditions, and more. 

Reviewer: ```suggestion
WebSockets are based on persistent and stateful connections. A connection can abnormally close in the lapse between when it was established and its use in an operation due to factors like idle timeouts, network conditions, and more. 
```

= WebSocket Reconnection
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: connectors, ws, wss, websocket, reconnection

> WebSockets are based on persistent and stateful connections. Therefore, it is possible for a connection to be abnormally closed in the lapse between it was established and it's being used in any operation due to factors like idle timeouts, network conditions, etc. 

Reviewer: ```suggestion
WebSockets are based on persistent and stateful connections. Therefore, it is possible for a connection to abnormally close in the lapse between the time it was established and the time it is used in any operation due to factors like idle timeouts, network conditions, and so on. 
```

> Inbound sockets cannot be automatically reconnected because for those connections to be reestablished because it is up to the remote system to resend the connection request.

Reviewer: ```suggestion
Inbound sockets cannot be automatically reconnected because the remote system must resend the connection request for the connections to be reestablished.
